### DISCUSSION TITLE
Merging era5_land and era5_single netcdf files

### DISCUSSION THREAD

**Discussion by abdulrehman565** [Deltares/hydromt#361] [Q&A] [answered]
I am taking the pressure, kin, kout from era5_single product from the following link
https://cds.climate.copernicus.eu/cdsapp#!/dataset/reanalysis-era5-single-levels?tab=overview

and I am taking precipitation and temperature data from era5_land product 
https://cds.climate.copernicus.eu/cdsapp#!/dataset/reanalysis-era5-land?tab=overview

The precipitation data worked but it gave an error during the calculation of temp_pet_fn. (attached below)
Is there a way to merge era5_land and era5_single file so that it works correctly?
![WhatsApp_Image_2023-05-02_at_4 51 27_PM](https://github.com/Deltares/hydromt/assets/130044572/d54dac63-876e-4caf-85cc-60c256a25f35)

**Comment by hboisgon** [comment, marked as answer]
For forcing, precipitation and temperature can come from a different data source.
To compute pet, all required variables should come from the same single data source {"temp", "press" or "press_msl", "kin", "kout"}. Else you have some pre-processing to do on your own before using hydromt to compute PET but I would really recommend using the same data source
